### PR TITLE
Nomis/dsos 1915/improve t1 weblogic monitoring

### PR DESCRIPTION
**Pull request by robertsweetman**
healthcheck.sh removes keepalive.htm if one/many weblogic related services are missing
healthcheck.sh checks if there's a weblogic service issue every 120 seconds 
logging records keepalive.htm addition/removal

### CODE REVIEW

File: ansible/roles/collectd/vars/main.yml [hunk 1]
```diff
@@ -8,4 +8,4 @@ service_list_oracle: "oracle-ohasd oracleasm"
 
 # service lists vars
 service_list_linux: "chronyd sshd"
-service_list_weblogic: "weblogic-node-manager" # weblogic-server might also be needed to be added here
+service_list_weblogic: "weblogic-node-manager weblogic-healthcheck" # weblogic-server might also be needed to be added here
```
Comment (drobinson-moj) [inline ansible/roles/collectd/vars/main.yml:11]: I don't think we need weblogic-node-manager in this list.  Just weblogic-healthcheck will be enough.

**Review by drobinson-moj — CHANGES_REQUESTED** [review]

File: ansible/roles/nomis-weblogic/tasks/install-healthcheck.yml [hunk 2]
```diff
@@ -32,4 +44,4 @@
       changed_when: false
 
   # block
-  when: not weblogic_healthcheck_installed_check.stat.exists
+  # when: not weblogic_healthcheck_installed_check.stat.exists
```
Comment (drobinson-moj) [inline ansible/roles/nomis-weblogic/tasks/install-healthcheck.yml:47]: Enable healthcheck needs to be enabled as well

File: ansible/roles/nomis-weblogic/templates/10.3/etc/init.d/weblogic-healthcheck [hunk 3]
```diff
@@ -1,45 +1,90 @@
 #!/bin/bash
 #
 # weblogic-healthcheck        Control weblogic healthcheck file
-#
-# chkconfig: - 95 05
-# description: weblogic server
```
Comment (drobinson-moj) [inline ansible/roles/nomis-weblogic/templates/10.3/etc/init.d/weblogic-healthcheck:4]: chkconfig and description lines need to be there.  It controls order things start up.  Fix description though.

File: ansible/roles/nomis-weblogic/tasks/install-healthcheck.yml (hunk 2 above)
Comment (robertsweetman) [reply ansible/roles/nomis-weblogic/tasks/install-healthcheck.yml]: Oops, indeed, forgot to revisit this file when I commented this step out in the debug process

**Review by drobinson-moj — APPROVED** [review]

**Pull request merged**